pom.xml

```diff
@@ -38,7 +39,7 @@
     <dep.maven.source.plugin.version>3.2.1</dep.maven.source.plugin.version>
     <dep.maven.surefire.plugin.version>2.22.2</dep.maven.surefire.plugin.version>
     <dep.scala.maven.plugin.version>3.4.4</dep.scala.maven.plugin.version>
-    <dep.scala.version>2.12.10</dep.scala.version>
+    <dep.scala.version>${scala.version}</dep.scala.version>
```

Comment:
Is this indirection from `dep.scala.version` to `scala.version` necessary? It seems odd to have two properties for the same thing.

Reply:
Maybe not, I can try removing it.